Freshsales Suite - traits is mandatory for Identify (#6681)
segmentio/segment-docs: src/connections/destinations/catalog/freshsales-suite-crm/index.md

#### Traits

> Traits are mandatory in an identify method, which contains pieces of information you know about a user.

Reviewer: ```suggestion
Traits are pieces of information you know about a user. They are a mandatory part of the [Identify method](/docs/connections/spec/identify/).
```
